new-findings.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+GO_SRC_PATH="$(go env GOPATH)/src"
+CURRENT_BRANCH=$(git branch --show-current)
```

Comment:
Nit: I think `FEATURE_BRANCH` would be a more accurate name

Reply:
Done.